### ISSUE TITLE
[Bug] Timezones do not work

### ISSUE THREAD

**Issue by ImDini** [TheCommieAxolotl/BetterDiscord-Stuff#18]
### Addon

Timezones

### What happened?

There is nothing new when opening profiles, even after discord restart.

**Comment by maenDisease** [comment]
Enable the plugin, right click any user pfp, and set a timezone to a user

**Comment by ggtylerr** [comment]
I'm not able to get it to work either, even with the above advice. Looking in the console, it seems there's an error upon opening the context menu: 
![image](https://user-images.githubusercontent.com/28498877/218349704-768649f0-46d8-48f9-8b3a-a866a4f57cf7.png)

<details>
<summary>Full error</summary>

```
bc619a8efeedbcb002b1.js:152 [BetterDiscord] [ContextMenu~runPatches] Could not run user-context patch for (ret, props) => {
                              ret.props.children[0].props.children.push([
                                  ContextMenu.buildItem({ type: "separator" }),
                           … TypeError: ret.props.children[0].props.children.push is not a function
    at eval (Timezones.plugin.js:115)
    at Function.runPatches (<anonymous>:4:405559)
    at i (<anonymous>:4:405106)
    at ya (8f641802d49465c1d02c.js:1608)
    at Ns (8f641802d49465c1d02c.js:1619)
    at Su (8f641802d49465c1d02c.js:1663)
    at Oc (8f641802d49465c1d02c.js:1655)
    at Ic (8f641802d49465c1d02c.js:1655)
    at hc (8f641802d49465c1d02c.js:1654)
    at oc (8f641802d49465c1d02c.js:1649)
    at g (8f641802d49465c1d02c.js:1772)
    at L (8f641802d49465c1d02c.js:1773)
    at t.exports (bc619a8efeedbcb002b1.js:21)
    at g.<computed> (bc619a8efeedbcb002b1.js:36)
    at Number.m (bc619a8efeedbcb002b1.js:36)
    at MessagePort.b (bc619a8efeedbcb002b1.js:36)
```

</details>

Unfortunately this is the only error I could find relating to the plugin. I can't even see any HTML tags that could be relating to it.

**Comment by JuniorGenius** [comment]
Same issue and errors here.
Discord PTB 173806, BD Core v1.8.4, ZLib 2.0.11.
Latest of all as well

**Comment by TheCommieAxolotl** [comment]
Fixed in [6c11b28](https://github.com/TheCommieAxolotl/BetterDiscord-Stuff/commit/6c11b281e85f110cf42976dd32cae262a9021e3e)
